java/src/main/native/src/row_conversion.cu

```diff
@@ -1918,19 +1865,16 @@ std::vector<std::unique_ptr<column>> convert_to_rows(
       });
 
   // blast through the entire table and convert it
-  dim3 const blocks(
-      util::div_rounding_up_unsafe(gpu_tile_infos.size(), NUM_TILES_PER_KERNEL_TO_ROWS));
+  dim3 const blocks(gpu_tile_infos.size());
   dim3 const threads(NUM_THREADS);
 
   // build validity tiles for ALL columns, variable and fixed width.
   auto validity_tile_infos = detail::build_validity_tile_infos(
       tbl.num_columns(), num_rows, shmem_limit_per_tile, batch_info.row_batches);
 
   auto dev_validity_tile_infos = make_device_uvector_async(validity_tile_infos, stream);
-  dim3 const validity_blocks(
-      util::div_rounding_up_unsafe(validity_tile_infos.size(), NUM_VALIDITY_TILES_PER_KERNEL));
-  dim3 const validity_threads(
-      std::min(validity_tile_infos.size() * NUM_VALIDITY_THREADS_PER_TILE, 128lu));
+  dim3 const validity_blocks(validity_tile_infos.size());
```

Comment:
Since these kernels are fundamentally warp-oriented, it might make sense to express this stuff in terms of just # of blocks and # of warps per block and leave "threads" out of it for easier reader comprehension.